austation/code/modules/antagonists/bloodsucker/datum_hunter.dm

```diff
@@ -0,0 +1,307 @@
+		to_chat(owner, "<span class='warning'>You detect signs of monsters to the <b>[dir2text(get_dir(my_loc,get_turf(scan_target)))]!</b></span>")
+	// Will yield a "?"
+	else
+		to_chat(owner, "<span class='notice'>There are no monsters nearby.</span>")
+	// Force Point-To Immediately
+	point_to_target()
+
+/datum/status_effect/agent_pinpointer/hunter_edition/scan_for_target()
+	// Lose target? Done. Otherwise, scan for target's current position.
+	if (!scan_target && owner)
+		owner.remove_status_effect(/datum/status_effect/agent_pinpointer/hunter_edition)
+
+	// NOTE: Do NOT run ..(), or else we'll remove our target.
+
+
+/datum/status_effect/agent_pinpointer/hunter_edition/Destroy()
+	if (scan_target)
+		to_chat(owner, "<span class='notice'>You've lost the trail.</span>")
+	..()
+*/
+
+
+
+
+
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+
+
+/datum/action/bloodsucker/trackvamp/
+	name = "Track Monster"//"Cellular Emporium"
+	desc = "Take a moment to look for clues of any nearby monsters.<br>These creatures are slippery, and often look like the crew."
+	button_icon = 'austation/icons/Fulpicons/fulpicons.dmi'	//This is the file for the BACKGROUND icon
+	background_icon_state = "vamp_power_off"		//And this is the state for the background icon
+	icon_icon = 'austation/icons/Fulpicons/fulpicons.dmi'		//This is the file for the ACTION icon
+	button_icon_state = "power_hunter" 				//And this is the state for the action icon
+
+	// Action-Related
+	amToggle = FALSE
+	cooldown = 200 // 10 ticks, 1 second.
+	bloodcost = 0
+
+
+
+/datum/action/bloodsucker/trackvamp/ActivatePower()
+
+	var/mob/living/user = owner
+
+	to_chat(user, "<span class='notice'>You look around, scanning your environment and discerning signs of any filthy, wretched affronts to the natural order.</span>")
+
+	if (!do_mob(user,owner,80))
+		return
+
+	// Add Power
+	// REMOVED //user.apply_status_effect(/datum/status_effect/agent_pinpointer/hunter_edition)
+	// We don't track direction anymore!
+
+	// Return text indicating direction
+	display_proximity()
+
+	// NOTE: DON'T DEACTIVATE!
+	//DeactivatePower()
+
+
+
+/datum/action/bloodsucker/trackvamp/proc/display_proximity()
+	// Pick target
+	var/turf/my_loc = get_turf(owner)
+	//var/list/mob/living/carbon/vamps = list()
+	var/best_dist = 9999
+	var/mob/living/best_vamp
+
+	// Track ALL MONSTERS in Game Mode
+	var/list/datum/mind/monsters = list()
+	monsters += SSticker.mode.bloodsuckers
+	monsters += SSticker.mode.devils
+	monsters += SSticker.mode.cult
+	monsters += SSticker.mode.wizards
+	monsters += SSticker.mode.apprentices
+	//monsters += SSticker.mode.servants_of_ratvar
+	monsters += SSticker.mode.changelingsgm
```

Comment:
action name: trackvamp
what it does: tracks vamps, devils, cults, wizards, wizard apprentices, and changelings

Reply:
Yeah that was mostly the plan. I think I might leave them in cause if someone does a dynameme round. Hunters can track other evils which might be cool. They can only spawn on a bloodsucker round though. 

I might make some changes to really EMPHASISE if hunters just tell the crew, hunters should be treated as class 4, aka against nanotraeson. Perhaps they snuck on board and aren't apart of the crew which forces them to keep it between THEM and the bloodsuckers/evils if it is a dynameme.

Reply:
I had already envisioned on commenting them out when I saw that code, so after some test merges if we see that hunters are actually spawning on normal rounds, I will go about disabling that. If they don't spawn, then seems fine to leave it in.